app/src/main/java/com/dsu/pocapplication/news/ui/login/fragments/LoginFragment.kt

```diff
@@ -0,0 +1,4 @@
+package com.dsu.pocapplication.news.ui.login.fragments
```

Comment:
there is no requirement of package "fragments" inside login

Reply:
done